models/users.js

```diff
@@ -119,29 +119,38 @@ const getSuggestedUsers = async (skill) => {
 
 /**
  * Fetches the data about our users
- * @param query { Object }: Filter for users
+ * @param query { search, next, prev, size, page }: Filter for users
  * @return {Promise<userModel|Array>}
  */
 const fetchUsers = async (query) => {
-  const appendUsernamePrefixQuery = (dbQuery) => {
-    return dbQuery
-      .orderBy("username")
-      .startAt(query.search.toLowerCase().trim())
-      .endAt(query.search.toLowerCase().trim() + "\uf8ff");
-  };
   try {
     // INFO: default user size set to 100
     // INFO: https://github.com/Real-Dev-Squad/website-backend/pull/873#discussion_r1064229932
     const size = parseInt(query.size) || 100;
-    const page = size * (parseInt(query.page) || 0);
-
-    let dbQuery = userModel.limit(size).offset(page);
+    const doc = (query.next || query.prev) && (await userModel.doc(query.next || query.prev).get());
+    let dbQuery = query.prev ? userModel.limitToLast(size) : userModel.limit(size);
     if (Object.keys(query).length) {
+      dbQuery = dbQuery.orderBy("username");
```

Comment:
Should we have the ordering by default for all queries

Reply:
yes